src/function.ts

```diff
@@ -39,26 +42,52 @@ export function isFunction<P = any, O = any>(a: any): a is IFunction<P, O> {
 
 export type AnyLambda = Function<any, any>;
 
+export interface FunctionEventBusTargetProps
+  extends Omit<aws_events_targets.LambdaFunctionProps, "event"> {}
+
 export type FunctionClosure<P, O> = (
   payload: P,
   context: Context
 ) => Promise<O>;
 
-export interface IFunction<P, O> {
+/**
+ * Wraps an {@link aws_lambda.Function} with a type-safe interface that can be
```

Comment:
Is it this interface that is doing the wrapping? Don't think that makes sense?

Reply:
Wonder if this was a merge issue. I don't remember this being there.